### ISSUE TITLE
Inline linked image: inline popovers overlap and can hardly be used

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#15761]
- add a paragraph block
- add an inline image within the paragraph
- link the image
- click or select the image
- both the image width inline popover and the image link popover appear at the same time
- the link popover overlaps the width popover:

<img width="1003" alt="Screenshot 2019-05-21 at 22 23 18" src="https://user-images.githubusercontent.com/1682452/58128510-1848e300-7c18-11e9-8544-7322b19989a6.png">

Besides the UI glitch, using the width popover is hardly doable even with the mouse.

Also, not clear to me how to move focus to the two popovers when using only the keyboard.

**Comment by draganescu** [comment]
As of now (Gutenberg 7.9.1) with the new link UI the width popover is totally hidden:

<img width="447" alt="Screenshot 2020-04-22 at 13 31 33" src="https://user-images.githubusercontent.com/107534/79971872-ac14d600-849d-11ea-8e75-c1d8371dd2c3.png">

**Comment by afercia** [comment]
Yup, can still reproduce as well on latest Gutenberg plugin. It's April 2020 now. Issue created on May 2019.

**Comment by ndiego** [comment]
This remains an issue in Gutenberg trunk. 

![image](https://user-images.githubusercontent.com/4832319/214336413-2af9275d-04a5-4fba-bb87-cca5df45b9f9.png)

**Comment by karmatosed** [comment]
This is still an issue:

<img width="546" alt="Image" src="https://github.com/user-attachments/assets/6c9fde19-ca46-4f88-b9b8-7eba98933328" />

**Comment by karmatosed** [comment]
I am removing the design feedback as this should just show one at a time, it to me doesn't need to wait for feedback as a result.
